docs/setup-vendor.md

```diff
@@ -0,0 +1,28 @@
+# Getting started with Private Packagist vendors on a Composer project
+
+You can start using Private Packagist from a vendor in either a new or an existing Composer project.
```

Comment:
This needs to explain what vendor is and why someone would want to run `composer init` or `composer create-project`. Similar to how the setup guide explains how to create a new Composer project with Private Packagist and then goes on to list the two options.

To make it clearer why someone might not need to run either of those two commands it also might make sense to show how to just add setup auth, add the repo to an existing composer.json and add a single package. As this might be how most vendor customers will use Private Packagist.

Reply:
ok, I tried restructuring this and added a section for installing a single package.